SlackPrNotification.ts

```diff
@@ -9,17 +9,17 @@ const authorName: string = process.env.PULL_REQUEST_AUTHOR_NAME;
 const authorIconUrl: string = process.env.PULL_REQUEST_AUTHOR_ICON_URL;
 const compareBranchOwner: string = process.env.PULL_REQUEST_COMPARE_BRANCH_OWNER;
 const compareBranchName: string = process.env.PULL_REQUEST_COMPARE_BRANCH_NAME;
+const baseRepoName: string = process.env.PULL_REQUEST_BASE_REPO_NAME;
 const baseBranchOwner: string = process.env.PULL_REQUEST_BASE_BRANCH_OWNER;
 const baseBranchName: string = process.env.PULL_REQUEST_BASE_BRANCH_NAME;
 const sendHereMention: string = process.env.IS_SEND_HERE_MENTION.toLowerCase() === "true" ? "<!here>\n" : "";
 
-const prFromFork: string = process.env.IS_PR_FROM_FORK;
-const compareBranchText: string = prFromFork === "true" ? compareBranchOwner + ":" + compareBranchName : compareBranchName;
-const baseBranchText: string = prFromFork === "true" ? baseBranchOwner + ":" + baseBranchName : baseBranchName;
-
 const makePretty: boolean = process.env.MAKE_PRETTY.toLowerCase() === "true"; //Priority is pretty > compact > normal
 const makeCompact: boolean = process.env.MAKE_COMPACT.toLowerCase() === "true";
 
+const baseBranchText: string = baseRepoName + ":" + baseBranchName;
```

Comment:
This is in a couple places, but I have two tidbits:

1. Personally, I think both base and compare should either have the owner and semi-colon prefix when it's relevant, or not at all; I'm not a huge fan of always printing the baseRepoName in the baseBranchText, but that could just be me and my pedanticness.
2. Why is the baseRepoName needed at all? Couldn't baseBranchOwner just be used and avoid the extra variable?

FYI I have not tested this change myself.